src/Core/src/Platform/Windows/ButtonExtensions.cs

```diff
@@ -50,10 +50,10 @@ public static void UpdateStrokeThickness(this Button platformButton, IButtonStro
 
 		public static void UpdateCornerRadius(this Button platformButton, IButtonStroke buttonStroke)
 		{
-			var radius = buttonStroke.CornerRadius > -1 ? buttonStroke.CornerRadius : 0;
+			var radius = buttonStroke.CornerRadius;
 
 			if (radius >= 0)
-				platformButton.Resources.SetValueForAllKey(CornerRadiusResourceKeys, WinUIHelpers.CreateCornerRadius(radius));
+				platformButton.Resources.SetValueForAllKey(CornerRadiusResourceKeys, WinUIHelpers.CreateCornerRadius(buttonStroke.CornerRadius));
```

Comment:
Should we use the radius field instead of getting the value again?

Reply:
Addressed!